__tests__/app.test.js

```diff
@@ -137,3 +137,45 @@ describe("GET /api/articles", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id/comments", () => {
+  test("sends an array of comments belonging to a single article to the client", () => {
+    return request(app)
+    .get("/api/articles/1/comments")
+    .expect(200)
+    .then((response) => {
+      const {body} = response;
+      const {comments} = body;
+      expect(comments.length).toBe(11)
+      comments.forEach((comment) => {
```

Comment:
how do we know that all of these comments are the correct ones? "belonging to a single article" 👀